packages/google_maps_flutter/google_maps_flutter_web/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ flutter:
         fileName: google_maps_flutter_web.dart
 
 dependencies:
+  collection: ^1.16.0
```

Comment:
I don't think we need this just to insert in the z-index sorted list (I'll take a look!)

Reply:
(This is already used by the platform_interface, so it can stay)